app/views/ChooseProvider.js

```diff
@@ -485,7 +485,7 @@ const styles = StyleSheet.create({
   textInput: {
     marginLeft: 10,
   },
-  autoSubcribe: {
+  autoSubscribe: {
```

Comment:
ideally don't include code changes in language changes. But I'll allow it for this time